mpas_analysis/shared/generalized_reader/generalized_reader.py

```diff
@@ -124,9 +130,35 @@ def open_multifile_dataset(fileNames, calendar, simulationStartTime=None,
                                  startDate=startDate,
                                  endDate=endDate)
 
-    ds = xarray.open_mfdataset(fileNames,
-                               preprocess=preprocess_partial,
-                               decode_times=False, concat_dim='Time', **kwargs)
+    # we don't want to decode times, since we're typically using dates
+    # outside the allowed range
+    kwargs['decode_times'] = False
+    # files differ along the Time dimension
+    kwargs['concat_dim'] = 'Time'
```

Comment:
Moved these here in case a user sets `decode_times` or `concat_dims` in the `kwargs` dictionary.  I overwrite them without checking for existing values because the code really won't work if the user sets them to wrong values.  I don't think there needs to be a warning about this because setting them explicitly is a bit weird anyway...